<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uC Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>228</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainSpacer">
     <item>
      <layout class="QFormLayout" name="ConnInfoLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="DeviceLabel">
         <property name="text">
          <string>Device:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="DeviceCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ConnTypeLabel">
         <property name="text">
          <string>Connection Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ConnTypeCombo"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="SpeedLabel">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="SpeedCombo"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ConnInfoEditLabel">
         <property name="text">
          <string>Connection Info:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="ConnInfoEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="GUITypeLabel">
         <property name="text">
          <string>GUI Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="GUITypeCombo"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="DeviceConnect">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
